aspnetcore/fundamentals/localization.md

```diff
@@ -320,4 +320,4 @@ Terms:
 
 * [Localization.StarterWeb project](https://github.com/aspnet/entropy) used in the article.
 * [Resource Files in Visual Studio](https://docs.microsoft.com/cpp/windows/resource-files-visual-studio)
-* [Resources in .resx Files](https://docs.microsoft.com/dotnet/framework/resources/working-with-resx-files-programmatically)
\ No newline at end of file
+* [Resources in .resx Files](https://docs.microsoft.com/dotnet/framework/resources/working-with-resx-files-programmatically)
```

Comment:
What's changed here?!!

Reply:
It's a difference in white space.

Reply:
I didn't notice anything 😄

Reply:
@hishamco  Yeah I wasn't sure what changed there either haha